trustpoint/pki/views/issuing_cas.py

```diff
@@ -25,13 +29,18 @@ class IssuingCaContextMixin(TpLoginRequiredMixin, ContextDataMixin):
     context_page_category = 'pki'
     context_page_name = 'issuing_cas'
 
-
-class IssuingCaTableView(IssuingCaContextMixin, TpLoginRequiredMixin, SingleTableView):
+# @TODO: Remove Security Level Restriction from IssuingCaTableView. Only for demo purpose
+class IssuingCaTableView(SecurityLevelMixin, IssuingCaContextMixin, TpLoginRequiredMixin, SingleTableView):
     """Issuing CA Table View."""
 
     model = IssuingCaModel
     table_class = IssuingCaTable
     template_name = 'pki/issuing_cas/issuing_cas.html'
+    security_feature = SecurityFeatures.ISSUING_CA_TABLE_VIEW
+    no_permisson_url = '/home/dashboard/'
```

Comment:
Would you agree in renaming this to `disabled_by_security_level_url`? `no_permission_url` implies user-level permission for me personally. Unless we want to implement user-level access permissions at a later time and use the same URL for both security level and user role based redirects.

Reply:
Fully.
As you said:
There are only two hard things in Computer Science: cache invalidation and naming things.

Done